### DISCUSSION TITLE
Vue 3, Electron 13, vue-cli-plugin-electron-builder v2.0.0

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [nklayman/vue-cli-plugin-electron-builder#1329] [General]
Hi @nklayman do you have a date for the v2.0.0 release planned?

Also, are you planning to add support for Vue 3 and Electron 13 in v.2.0.0, or do you think it's better to wait for them to enter stable channels first?

**Comment by nklayman** [comment]
Vue 3 is already supported and should work just fine. I won't be officially supporting Electron 13 until it is stable, but there is a good chance it will work just fine.

**Reply by aleksey-hoffman** [reply]
Hi @nklayman remember I told you I'm developing a file manager app using this plugin? You asked to remind you when I release it. I just published the first release if you want to check it out 😊
https://github.com/aleksey-hoffman/sigma-file-manager

**Reply by ajmasia** [reply]
> Vue 3 is already supported and should work just fine. I won't be officially supporting Electron 13 until it is stable, but there is a good chance it will work just fine.

Do you plan to use vite? 😄 
Thanks!

**Reply by nklayman** [reply]
I can only use Vite if Vue CLI switches to it. Otherwise I'm stuck with webpack.
